packages/tracing/src/span.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable max-lines */
-import { getCurrentHub } from '@sentry/hub';
-import { Hub, Primitive, Span as SpanInterface, SpanContext, TraceHeaders, Transaction } from '@sentry/types';
+import { getCurrentHub, Hub } from '@sentry/hub';
+import { Primitive, Span as SpanInterface, SpanContext, TraceHeaders, Transaction } from '@sentry/types';
```

Comment:
It actually is, though. The change here is from importing the type to importing the class. Later on in the file I'm calling `Hub.getScope()`, a method which exists on the class but isn't in the type. The alternative to making this change is to add it to the type, which I'm happy to do instead if you think that's better.

Reply:
Understood, it's probably me (I'm not so familiar with this code) but it further raises the question why we don't have `getScope` in the interface ( since we do have `pushScope`, `popScope`, `withScope`). 

Anyway, not the right place to discuss general architecture of the API.